indico/modules/rb_new/client/js/modules/admin/AdminEquipmentTypes.jsx

```diff
@@ -0,0 +1,374 @@
+/* This file is part of Indico.
```

Comment:
Maybe split this file in 3? There are 3 components anyway. Or at least in 2.

Reply:
good idea